packages/api/src/controllers/region.js

```diff
@@ -0,0 +1,63 @@
+import Router from 'express/lib/router'
+import { authMiddleware } from '../middleware'
+import { db } from '../store'
+
+const app = Router()
+
+app.get('/', authMiddleware({ admin: true }), async (req, res, next) => {
+  const [regions, cursor] = await db.region.find()
+  if (req.query.grouped) {
```

Comment:
Hmm, this will show grouped results for `?grouped=true` and `?grouped=false` and `?grouped=potato`. Maybe 
```suggestion
  if (req.query.grouped === "true") {
```